kernel/yosys.cc

```diff
@@ -318,8 +318,10 @@ int run_command(const std::string &command, std::function<void(const std::string
 #else
 	return WEXITSTATUS(ret);
 #endif
-}
+#else
+	log_cmd_error("Command execution is disabled in this build.\n");
```

Comment:
Make it a `log_warn` please since `log_cmd_error` throws an exception if I am not mistaken, and the call site might not be prepared to handle that. Also rephrase to "Shell command execution ...".